triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceAppConf.scala

```diff
@@ -110,6 +111,7 @@ private[trigger] final case class TriggerServiceAppConf(
     triggerConfig: TriggerRunnerConfig = DefaultTriggerRunnerConfig,
     rootLoggingLevel: Option[Level] = None,
     logEncoder: LogEncoder = LogEncoder.Plain,
+    metrics: Option[MetricsConfig] = None,
```

Comment:
Nit: `metrics` is a pretty generic term, hence naming this as `metricsConfig` should improve readability.

Reply:
It is the name of the key to used in the config file. Using `metrics` aligns with config files of the other components (json-api, canton components).
UX trumps code readability in this case.